pilot/model/BUILD

```diff
@@ -27,6 +26,8 @@ go_library(
         "@io_istio_api//mixer/v1:go_default_library",
         "@io_istio_api//mixer/v1/config/client:go_default_library",
         "@io_istio_api//routing/v1alpha1:go_default_library",
+        "@io_istio_api//routing/v1alpha2:go_default_library",
+        "//pilot/model/test:go_default_library",
```

Comment:
Is this the correct ordering? Do we have a lint tool for BUILD files to automatically correct this?

Reply:
I don't know. I think Intellij automatically re-ordered them, but I don't know if they were in any sort of ordering before.